.github/workflows/publish_canary_docker.yml

```diff
@@ -23,5 +22,5 @@ jobs:
     - name: tag & publish
       run: |
         ./scripts/tag_image_and_push.sh playwright-java:localbuild-focal playwright.azurecr.io/public/playwright/java:next
-        ./scripts/tag_image_and_push.sh playwright-java:localbuild-focal playwright.azurecr.io/public/playwright/java:next-focal
+        ./scripts/tag_image_and_push.sh playwright-java:localbuild-focal playwright.azurecr.io/public/playwright/java:focal-next
```

Comment:
Revert this? According to https://mcr.microsoft.com/v2/playwright/tags/list we use *-<pw version>-<distro version> scheme, let's stick with that.

Reply:
you are right, this change was not correct. Good catch.